drivers/i2c/i2c_nrfx_twi.c

```diff
@@ -90,10 +94,6 @@ static int i2c_nrfx_twi_configure(struct device *dev, u32_t dev_config)
 {
 	nrfx_twi_t const *inst = &(get_dev_config(dev)->twi);
 
-	if (I2C_ADDR_10_BITS & dev_config) {
-		return -EINVAL;
```

Comment:
I think this should stay. As long as the flag is available in the API, though it is marked as deprecated, it should be handled correctly to keep the backward compatibility.

Reply:
Added it back